### PR TITLE
Experience/6523

### LINKED ISSUE #6523: Fetch Refactor: Replace NewApi: UseOrganizationResource
## Problem statement

As a report stream developer I want to replace all instances of usage of NewApi and useRequestConfig with react query based hooks.

This ticket accomplishes replacement for files downstream of UseOrganizationResource:
* UseOrgName
    * Daily
    * Summary
    * Submissions

## What you need to know

 [Links to documents, workflows, or list of key people and contact info]

- [Document 1](https://...)
- [Document 2](https://...)
- ...

## Acceptance criteria

- [ ] usage of NewApi and useRequestConfig on all mentioned files is replaced with new react query based functionality

## To do

- [ ] ...

### PR DESCRIPTION
**Pull request by jpandersen87**
This PR removes usage of NewApi/UseOrganizationResource/UseOrgName with a new UseOrganizationSettings hook that relies on useAuthorizedFetch (based on react-query).

Test Steps:
1. Log into reportstream
2. Set simple_report from organization list
3. Go to Submissions page
4. Verify organization description near page header
5. Go to Upload & Validate > Validate page
6. Verify organization description near page header
7. Go to Daily Data page
8. Verify organization description near page header

## Changes
- Created UseOrganizationSettings hook with test that relies on useAuthorizedFetch (react-query)
- Replaced usage of NewApi/UseOrganizationResource/UseOrgName with UseOrganizationSettings
- Marked older functions as deprecated

### Testing
- [x] Tested locally?
- [x] Ran `./prime test` or `./gradlew testSmoke` against local Docker ReportStream container?
- [x] (For Changes to /frontend-react/...) Ran `npm run lint:write`? 
- [x] Added tests?

### Process
- [ ] Are there licensing issues with any new dependencies introduced?
- [x] Includes a summary of what a code reviewer should test/verify?
- [ ] Updated the release notes?
- [ ] Database changes are submitted as a separate PR?
- [ ] DevOps team has been notified if PR requires ops support?

## Linked Issues
- Fixes #6523 

## To Be Done
*Create GitHub issues to track the work remaining, if any*
- #6527

### CODE REVIEW

File: frontend-react/src/components/FileHandlers/FileHandler.tsx [hunk 1]
```diff
@@ -109,8 +109,8 @@ const FileHandler = ({
     }, [localError]);
 
     const { activeMembership } = useSessionContext();
-    const { organization, loading: organizationLoading } =
-        useOrganizationResource();
+    const { data: organization, isLoading: organizationLoading } =
```
Comment (kevinhaube) [inline frontend-react/src/components/FileHandlers/FileHandler.tsx:112]: Do. you mind adding a comment in here about moving off of `isLoading` and onto using the `Suspense` component where possible? Our React Query client is set up to use Suspense and ErrorBoundaries for loading and catching.

**Review by kevinhaube — APPROVED** [review]
🚀 LGTM!

File: frontend-react/src/hooks/UseOrgName.ts [hunk 2]
```diff
@@ -1,5 +1,8 @@
 import { useOrganizationResource } from "./UseOrganizationResource";
 
+/**
+ * @deprecated Please use useOrganizationSettins instead
```
Comment (penny-lischer) [inline frontend-react/src/hooks/UseOrgName.ts:4]: Change useOrganizationSettins to useOrganizationSettings

File: frontend-react/src/components/FileHandlers/FileHandler.tsx (hunk 1 above)
Comment (jpandersen87) [reply frontend-react/src/components/FileHandlers/FileHandler.tsx:113]: Made a comment. Let me know if the message is proper.

Comment (kevinhaube) [reply frontend-react/src/components/FileHandlers/FileHandler.tsx:113]: perfect! thank you :)

**Pull request merged**